gips/0064-graph-node-version-updates.md

```diff
@@ -0,0 +1,33 @@
+---
+GIP: 0064
+Title: Delegation of responsibility for Graph Node version updates
+Authors: Adam Fuller
+Created: 2024-02-08
+Updated: 2024-02-08
+Stage: Draft
+Category: Process
+---
+
+# Abstract
+
+The Graph Council should delegate responsibility for specification of the protocol-supported version of Graph Node to the Graph Node development team.
+
+# Motivation
+
+This GIP seeks to simplify the release process, and reduce bureaucracy, in order to reduce the time it takes for new features and fixes to be supported on the network by indexers.
+
+# Prior Art
+
+Currently, as specified in GIP-0008, the Graph Council ratifies each Graph Node version via a GGP.
```

Comment:
a reference to the actual ratified process may be appreciated https://snapshot.org/#/council.graphprotocol.eth/proposal/0xbdd884654a393620a7e8665b4289201b7542c3ee62becfad133e951b0c408444

Reply:
Good idea